games-puzzle/icebreaker/icebreaker-1.9.7.recipe

```diff
@@ -0,0 +1,49 @@
+SUMMARY="A classic puzzle game"
+DESCRIPTION="So, uh, there's a bunch of penguins on an iceberg in \
+Antarctica. You have been selected to catch them so they can be shipped \
+to Finland, where they are essential to a secret plot for world domination."
+HOMEPAGE="https://mattdm.org/icebreaker/"
+COPYRIGHT="2000-2002 Matthew Miller"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://mattdm.org/icebreaker/1.9.x/icebreaker-1.9.7.tgz"
+CHECKSUM_SHA256="ca8b344a6a37b2041fba83271ae5e7566cbc8dc4c3367addd46f2a2f3573017e"
+GLOBAL_WRITABLE_FILES="settings/icebreaker.scores keep-old"
+
+ARCHITECTURES="x86_gcc2 x86 ?x86_64"
+
+PROVIDES="
+	icebreaker = $portVersion
+	app:IceBreaker = $portVersion
+	"
+REQUIRES="
+	haiku
+	lib:libsdl
+	lib:libsdl_mixer
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	devel:libsdl
+	devel:libsdl_mixer
+	"
+BUILD_PREREQUIRES="
+	cmd:awk
+	cmd:gcc
+	cmd:make
+	"
+
+BUILD()
+{
+	make prefix=$prefix bindir=$appsDir datadir=$dataDir \
+		highscoredir=$settingsDir mandir=$manDir
+}
+
+INSTALL()
+{
+	make prefix=$prefix bindir=$appsDir datadir=$dataDir \
+		highscoredir=$settingsDir mandir=$manDir install
+	mv $appsDir/icebreaker $appsDir/IceBreaker
+
+	addAppDeskbarSymlink $appsDir/IceBreaker IceBreaker
```

Comment:
Oops, my mistake. You are perfectly right. Believe me or not, I had not noticed that all the apps in **`/boot/system/apps/`** have capital letters!.

This means it is perfectly fine to keep the *`mv $appsDir/icebreaker $appsDir/IceBreaker`**.

BTW, when you symlink something with **`addAppDeskbarSymlink`** you can ommit the second parameter if the name of the symlink is the same as that of the target. So, if you want, you can replace:
```
addAppDeskbarSymlink $appsDir/IceBreaker IceBreaker
```
by
```
addAppDeskbarSymlink $appsDir/IceBreaker
```
(I discovered that once when I had a look at a file called **`wrapper-script`** that haikuporter puts in the work-* subdirectory, **`work-1.9.7-1`** when you build icebreaker-1.9.7.  BTW, I browse this **`wrapper-script`** file very often.)